spikeextractors/extractors/mdaextractors/mdaextractors.py

```diff
@@ -188,7 +188,7 @@ def __init__(self, file_path, sampling_frequency=None):
         for unit_id in self._unit_ids:
             inds = np.where(self._labels == unit_id)
             max_channels = self._max_channels[inds].astype(int)
-            self.set_unit_property(unit_id, 'max_channel', max_channels[0])
+            self.set_unit_property(unit_id, 'mda_max_channel', max_channels[0])
```

Comment:
@magland is this change ok with you? I'll merge this and will revert it if it's a problem as I have another PR ins ST pending :)